src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +24,25 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    public List<LineAndStationResponse> findAllLines() {
+        List<Line> stations = lineRepository.findAll();
+        return stations.stream()
+            .map(LineAndStationResponse::of)
+            .collect(Collectors.toList());
+    }
+	public LineAndStationResponse findByLine(Long id) {
+		return LineAndStationResponse.of(lineRepository.findById(id).orElseThrow(()
+			-> new IllegalArgumentException("조회 할 수 없습니다.")));
```

Comment:
```suggestion
		return lineRepository.findById(id)
		        .map(LineAndStationResponse::of)
		        .orElseThrow(()-> new IllegalArgumentException("조회 할 수 없습니다."));
```
이렇게 표현할 수 있습니다.
그리고 CustomException을 만들어주면 어떨까요?
```suggestion
		return lineRepository.findById(id)
		        .map(LineAndStationResponse::of)
		        .orElseThrow(NotFoundLineException::new);
```